src/test/java/io/stargate/sgv2/jsonapi/api/v1/CountIntegrationTest.java

```diff
@@ -137,6 +147,7 @@ public void noFilter() {
           .then()
           .statusCode(200)
           .body("status.count", is(5))
+          .body("status.moreData", is(true))
```

Comment:
Would be nice to mention in test class Javadoc that the default settings are overridden by DseTestResource, so max count is 5, max count page size 2? (was first surprised by this but it makes sense with overrides)